src/components/VAudioTrack/VAudioTrack.vue

```diff
@@ -285,7 +285,9 @@ export default defineComponent({
 
     /* Timekeeping */
 
-    const duration = computed(() => (props.audio?.duration ?? 0) / 1e3) // seconds
+    const duration = computed(
+      () => (localAudio?.duration ?? props.audio?.duration ?? 0) / 1e3 // seconds
```

Comment:
This looks great! There's also another component that will need this update: `VGlobalAudioTrack`.

You can see this component on the audio search results page when your viewport is small, like on a mobile device (or just a narrow desktop browser window).

Reply:
Updated in `f3aa9f2`.

Hope its okay I added this to the single commit. Figured it was the same exact fix.